Server_Side_TLS.mediawiki

```diff
@@ -262,15 +262,12 @@ MBYCEQCHU6UNZoHMF6bPtj21Hn/bAgEC.....
 </source>
 
 == Pre-defined DHE groups ==
-In order to lower the burden of system administrators, several servers provide pre-computed DH groups. Unfortunately, the [https://weakdh.org/ logjam] report showed that it is very likely that a state-level adversary may have broken the most widely used 1024-bit DH group, Oakley group 2, standardized in [https://tools.ietf.org/html/rfc2409#section-6.2 rfc2409]].
+In order to lower the burden of system administrators, several servers provide pre-computed DH groups. Unfortunately, the [https://weakdh.org/ logjam] shows that pre-defined 1024 bits DHE is not safe, and administrators should either:
```

Comment:
I don't think this text is quite right either.

There are two forms of attack to be concerned with:
1. Passive
2. Active

It's true that there's a precomputation *passive* attack on 1024-bit DHE but that's not shown by logjam; it's been known forever. What logjam shows is that there's an active attack even if the pairs prefer a stronger group, but then it's not clear how much the precomputation buys you because active attacks are generally targeted and if you can do the precomputation attack on 1024-bit DHE at all, then you could at least potentially do it for a few high-value targets (the ones you are likely to mount active attacks on).